dev-packages/node-integration-tests/suites/express/multiple-init/server.ts

```diff
@@ -54,15 +54,19 @@ app.get('/test/error/:id', (req, res) => {
 
   setTimeout(() => {
     // We flush to ensure we are sending exceptions in a certain order
-    Sentry.flush(3000).then(
+    Sentry.flush(1000).then(
       () => {
+        // We send this so we can wait for this, to know the test is ended & server can be closed
+        if (id === '3') {
+          Sentry.captureException(new Error('Final exception was captured'));
```

Comment:
this lead to a "race condition" in tests, which did not fail the test, but showed errors in the test log:

we wait for the error event to be sent, then end the test. at this point the server is shut down, which leads to the http request being aborted. 

now, we specifically wait for this event, so we know everything is done.

Reply:
oh, I think my mistake here was that I though the test was terminated when both the request ends and the error was received. Maybe that is actually what we wanna do in the test? idk